src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -355,24 +331,19 @@ object MarathonSchedulerActor {
   }
 
   sealed trait Event
-  case class RunSpecScaled(runSpecId: PathId) extends Event
   case object TasksReconciled extends Event
   case class DeploymentStarted(plan: DeploymentPlan) extends Event
   case class DeploymentFailed(plan: DeploymentPlan, reason: Throwable) extends Event
   case class DeploymentFinished(plan: DeploymentPlan) extends Event
   case class TasksKilled(runSpecId: PathId, taskIds: Seq[Instance.Id]) extends Event
   case class CommandFailed(cmd: Command, reason: Throwable) extends Event
+  case object Noop extends Event // A stub for the fact that some commands do not send events
```

Comment:
what if we instead modeled Command different; `CommandWithAnswer` and `Command`?

Reply:
Just wait - I have a plan to remove those new commands altogether.